src/components/connection_handler/src/connection_handler_impl.cc

```diff
@@ -802,6 +802,7 @@ void ConnectionHandlerImpl::CloseSession(ConnectionHandle connection_handle,
       for (;service_list_itr != service_list.rend(); ++service_list_itr) {
         const protocol_handler::ServiceType service_type =
             service_list_itr->service_type;
+
```

Comment:
I think it's not good practice to add unnecessary files to pull request because of such changes